apps/ledger-live-desktop/src/renderer/families/evm/StakeFlowModal/EthStakingModalBody.tsx

```diff
@@ -39,7 +39,10 @@ export function EthStakingModalBody({ source, onClose, account, providers }: Pro
         pathname: value,
         ...(customDappUrl ? { customDappUrl } : {}),
         state: {
-          accountId: account.id,
+          accountId:
+            manifest?.params && "dappUrl" in manifest.params
+              ? getWalletApiIdFromAccountId(account.id)
+              : account.id,
```

Comment:
Shouldn't we reverse this condition and only send the LL account id for dapp browser v3 apps ?
```suggestion
            manifest?.dapp
              ? account.id
              : getWalletApiIdFromAccountId(account.id),
```